src/python/pants/engine/target.py

```diff
@@ -1536,6 +1536,45 @@ def compatible_with_sources_field(valid_type: Type[Sources]) -> bool:
     )
 
 
+# -----------------------------------------------------------------------------------------------
+# `Dependencies` field
+# -----------------------------------------------------------------------------------------------
+
+# NB: To hydrate the dependencies, use one of:
+#   await Get[Addresses](DependenciesRequest(tgt[Dependencies])
+#   await Get[Targets](DependenciesRequest(tgt[Dependencies])
+#   await Get[TransitiveTargets](DependenciesRequest(tgt[Dependencies])
```

Comment:
We can't put this in the docstring because it shows up in the output of `./pants target-types`.